test/ArgumentForwardingTests/ArgumentForwardingTests.cs

```diff
@@ -51,6 +51,7 @@ private void FindAndEnsureReflectorPresent()
         /// <param name="testUserArgument"></param>
         [Theory]
         [InlineData(@"""abc"" d e")]
+        [InlineData(@"""ábc"" d é")]
```

Comment:
Although I added this for quick and easy coverage, I think it'd be best to add an explicit unicode fact instead of putting it in this theory. Don't think the PR should block on this though.